### PR TITLE
fixed the last two attrs issue with overkill

### LINKED ISSUE #4180: Issues with the attrs reg
Have to address both before the final 2.0 release. 

Fix https://github.com/sunpy/sunpy/pull/3637#discussion_r426643732
Fix https://github.com/sunpy/sunpy/pull/3637#issuecomment-630089231

### PR DESCRIPTION
**Pull request by nabobalis**
Fixes #4180

### CODE REVIEW

File: sunpy/net/attr.py [hunk 1]
```diff
@@ -243,20 +227,28 @@ def update_values(cls, adict):
                             if len(pair) == 1:
                                 # Special case handling for * aka all values allowed.
                                 if pair[0] == "*":
-                                    strtonum
```
Comment (nabobalis) [inline sunpy/net/attr.py:246]: Woops.

File: docs/dev_guide/code_standards.rst [hunk 2]
```diff
@@ -52,15 +52,24 @@ Coding Style/Conventions
 Formatting
 ==========
 
-**This is optional and we do not enforce, check or expect you to do this.**
+We enforce a minimum level of code compliance and we have tools that will automate this step for you.
```
Comment (Cadair) [inline docs/dev_guide/code_standards.rst:55]:
```suggestion
We enforce a minimum level of code style and we have tools that will automate this step for you.
```
?

Comment (nabobalis) [reply docs/dev_guide/code_standards.rst]: Works for me

File: sunpy/net/attr.py [hunk 3]
```diff
@@ -243,20 +227,28 @@ def update_values(cls, adict):
...
-                            # This turns that digit into its name
-                            name = strtonum(pair[0][0])
-                            if pair[0][1:]:
-                                name = name + "_" + pair[0][1:]
+                        p = inflect.engine()
+                        # Sanitize part one: Check if the name is has a number in it
+                        if NUMBER_REGEX.match(pair[0]):
+                            # Now check if the entire name is a number
+                            if len(pair[0]) == NUMBER_REGEX.match(pair[0]).span()[1]:
+                                # This turns that digit into its name
```
Comment (Cadair) [inline sunpy/net/attr.py:241]:
```suggestion
                                # This turns that number into its name
```
Are we converting a digit (i.e. the firs one?) into a name or the whole number?

Comment (nabobalis) [reply sunpy/net/attr.py:241]: The entire number here.
Second if is just the digit.

**Pull request merged**